sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/CollectionPropertiesShouldBeReadOnly.cs

```diff
@@ -69,6 +70,15 @@ public class Message
         public List<int> Values { get; set; }
     }
 
+    // Ignore collections marked with Serializable attribute: https://github.com/SonarSource/sonar-dotnet/issues/2762
```

Comment:
I think it would be great to update also the [RSPEC](https://jira.sonarsource.com/browse/RSPEC-4004) and include the exception.

Reply:
Good point! I'll do that in #3146